pair-filter/pair_filter/image_chooser.py

```diff
@@ -144,10 +158,14 @@ def get_image_tag(self) -> Optional[str]:
         images = self.dockerhub_images.get(repo_name, '')
         if not images:
             return None
-        match = re.findall(pattern, images)
-        if len(match) == 0:
+        for image in images:
+            match = re.search(pattern, image)
+            if match is not None:
+                match_list.append(match)
+        if len(match_list) == 0:
```

Comment:
`if not match_list:` is more succinct and pythonic (`ctrl-f` "For sequences, (strings, lists, tuples), use the fact that empty sequences are false." in https://www.python.org/dev/peps/pep-0008/